lib/src/signed_video_h26x_auth.c

```diff
@@ -415,8 +400,7 @@ verify_hashes_with_hash_list(signed_video_t *self,
     // We do not know where in the sequence of NALUs they were lost. Simply add them before the
     // first item. If the first item needs a second opinion, that is, it has already been verified
     // once, we append that item. Otherwise, prepend it with missing items.
-    const bool append =
-        nalu_list->first_item->second_hash && !nalu_list->first_item->need_second_verification;
+    const bool append = nalu_list->first_item->second_hash;
```

Comment:
It might be better to do `const bool append = !!nalu_list->first_item->second_hash;`